src/main/scala/ai/privado/exporter/RepoConfigMetaDataExporter.scala

```diff
@@ -11,12 +11,16 @@ import ai.privado.model.Constants
 object RepoConfigMetaDataExporter {
 
   def getMetaData(cpg: Cpg, ruleCache: RuleCache) = {
-    val metaData   = mutable.LinkedHashMap[String, String]()
+    val metaData   = mutable.LinkedHashSet[mutable.HashMap[String, String]]()
     val configRule = ruleCache.getSystemConfigByKey(Constants.RepoPropertyConfig)
     try {
       val propertySources = cpg.property.filter(p => p.name matches configRule).l
       propertySources.foreach(p => {
```

Comment:
Also instead of using a `foreach` can we not use a `map`, that will be more scalish way of writing code

Reply:
yes done.